### ISSUE TITLE
Error in producing output with python3

### ISSUE THREAD

**Issue by mattiadg** [OpenNMT/OpenNMT-py#97]
I've just run a translation and I got this error. It seems that this part of code is only for python2. 
```bash
Traceback (most recent call last):
  File "translate.py", line 171, in <module>
    main()
  File "translate.py", line 120, in main
    for i in predBatch[b][0]]) + '\n')
  File "translate.py", line 120, in <listcomp>
    for i in predBatch[b][0]]) + '\n')
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by gladuo** [comment]
It's my fault. Seems not easily compatible for both py2 and py3.

**Comment by FrancisGregoire** [comment]
Removing .decode('utf-8') in line 119 fixed the issue for me.

**Comment by mattiadg** [comment]
Yes, it works also with python2  by simply removing .decode('utf-8')

**Comment by sebastianGehrmann** [comment]
Should be fixed with #109
